packages/service/README.md

```diff
@@ -184,6 +188,110 @@ pip install ni-measurement-plugin-sdk-generator
 
 ---
 
+## Generating Measurement Clients: Quick Start
```

Comment:
Why are we duplicating this whole section here? It isn't related to the 'service' in my mind, so I would just undo all these service readme changes.

Reply:
We should move this content into the -sdk-generator readme.

I guess the -sdk-generator readme could have a section on generating measurement plug-ins that links to the -sdk-service readme?

Reply:
Removed from -sdk-service readme and moved to -sdk-generator readme.
As per developing measurement plug-ins, created a section and linked it to the respective -sdk-service section.